### PR TITLE
Move EIP assets to assets folder

### PR DESCRIPTION
**Pull request by jacquesd**
After a [brief discussion](https://github.com/ethereum/EIPs/pull/907#issuecomment-377232154) with @Arachnid I suggest the following changes to the EIPs repository to make sure that EIPS with images and other assets pass the CI checks.

- Create `assets` folder
- Move existing EIPs (1, 107, 858) assets into the `assets` folder
- Update link to assets in EIPs 1, 107 and 858
- Describe the inclusion of assets for EIPs in `README.md`

> Note this is a change to the EIPs repository, not an EIP itself.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1 is in state Active, not Draft
 - File README.md is not an EIP
 - File assets/eip-1/process.png is not an EIP
 - File assets/eip-107/authorization-locked.png is not an EIP
 - File assets/eip-107/authorization-password.png is not an EIP
 - File assets/eip-107/authorization.png is not an EIP
 - File assets/eip-858/calculations.md is not an EIP
 - EIP 107 requires approval from one of (wighawag@gmail.com)
 - EIP 858 requires approval from one of (cslarson@gmail.com)

**Comment by jacquesd** [comment]
Note: the changes to EIP1, EIP107 and EIP858 are **solely to update the links to their respective images**. None of the actual content or wording has been modified!

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ EIPs can also be superseded by a different EIP, rendering the original obsolete.
 
 The possible paths of the status of EIPs are as follows:
 
-![EIP Process](eip-1/process.png)
+![EIP Process](/assets/eip-1/process.png)
```
Comment (Arachnid) [inline EIPS/eip-1.md]: Please don't use absolute paths - this will break for anyone running it on theirname.github.io.

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -114,7 +114,8 @@ Each EIP should have the following parts:
 EIP Formats and Templates
 -------------------------
 
-EIPs should be written in [markdown] format. Image files should be included in a subdirectory for that EIP.
+EIPs should be written in [markdown] format.
+Image files should be included in a subdirectory of the `assets` folder for that EIP as follow: `assets/eip-X` (for eip *X*).
```
Comment (Arachnid) [inline EIPS/eip-1.md]: Two * for bold.

Comment (jacquesd) [reply EIPS/eip-1.md]: I thought it looked nicer italics, but I'm happy to make it bold.

File: EIPS/eip-1.md (hunk 1 above)
Comment (jacquesd) [reply EIPS/eip-1.md]: ok I'll fix it to `../assets/eip-1/process.png` (and the other links too)

**Pull request merged**